library/bignum_core.h

```diff
@@ -129,6 +129,22 @@ size_t mbedtls_mpi_core_bitlen( const mbedtls_mpi_uint *A, size_t A_limbs );
 void mbedtls_mpi_core_bigendian_to_host( mbedtls_mpi_uint *A,
                                          size_t A_limbs );
 
+/** \brief         Compare a machine integer with an MPI.
```

Comment:
(minor) I find "machine integer" misleading - I expected to see `int min`. I think we normally just say "integer".

Reply:
“Integer” in this context could be an MPI. This one is a single limb, i.e. it's a machine-sized integer (perhaps longer than `int`, but no longer than `long long int`).

Reply:
I disagree (as you'd expect). "Compare an MPI with an integer" is exactly how (existing) `mbedtls_mpi_cmp_int()` describes itself.

(`mbedtls_mpi_core_clz()` calls such a thing (non-MPI) an "integer"; `mbedtls_mpi_core_mla()` calls it a "scalar"; `mbedtls_mpi_core_sub_int()` calls it a "scalar"; `mbedtls_mpi_lset()` calls it "an integer"; `mbedtls_mpi_add_int()` calls it "an integer"; `mbedtls_mpi_sub_int()` calls it "an integer" and so on.)

But you can keep it if you feel strongly enough about it